os/kernel/init/os_start.c

```diff
@@ -330,21 +330,25 @@ const struct tasklist_s g_tasklisttable[NUM_TASK_STATES] =
 	{                                              /* TSTATE_WAIT_MQNOTFULL */
 		&g_waitingformqnotfull,
 		TLIST_ATTR_PRIORITIZED
+	},
```

Comment:
line 321 should be applied like this.
Because signal also has the config to disable
```
#ifndef CONFIG_DISABLE_SIGNALS
volatile dq_queue_t g_waitingforsignal;
#endif
```

Reply:
line 175, g_assignedtasks should be in CONFIG_SMP